source/PlayerInfo.cpp

```diff
@@ -2704,7 +2707,7 @@ void PlayerInfo::CreateMissions()
 	// Sort missions on the job board alphabetically.
 	availableJobs.sort([](const Mission &lhs, const Mission &rhs)
 	{
-		return lhs.Name() < rhs.Name();
+		return lhs.IsService() ? (rhs.IsService() ? lhs.Name() < rhs.Name() : true) : lhs.Name() < rhs.Name();
```

Comment:
Does this put the services at the start of the list? I'd prefer them to be at the end.

Reply:
with the last commit they should be at the end

Reply:
Could you add it in the comment right above? (That it puts service missions at the end)